catalog/src/main/kotlin/com/adevinta/spark/catalog/model/Components.kt

```diff
@@ -146,11 +148,24 @@ private val Popovers = Component(
     description = R.string.component_popovers_description,
     guidelinesUrl = "$ComponentGuidelinesUrl/p/88a08c-popover/b/904ceb",
     docsUrl = "$PackageSummaryUrl/com.adevinta.spark.popover/index.html",
-    sourceUrl = "$SparkSourceUrl/kotlin/com/adevinta/popover/Color.kt",
+    sourceUrl = "$SparkSourceUrl/kotlin/com/adevinta/spark/components/popover/Popover.kt",
     examples = PopoverExamples,
     configurator = PopoverConfigurator,
 )
 
+private val BottomSheets = Component(
+    id = nextId(),
+    name = "BottomSheets",
+    illustration = R.drawable.bottomsheet,
+    tintIcon = false,
+    description = R.string.component_bottomsheets_description,
+    guidelinesUrl = "$ComponentGuidelinesUrl/p/67d41e-bottom-sheet/b/02056b",
+    docsUrl = "$PackageSummaryUrl/com.adevinta.spark.bottom-sheet/index.html",
```

Comment:
I think the url will not work as the package is `com.adevinta.spark.bottomsheet.modal`

Reply:
I've moved it to `com.adevinta.spark.components.bottomsheet`